SETUP.md

```diff
@@ -75,6 +75,23 @@ At this point you should be able to access the site via whatever webserver you c
 ---
 **Notes**
 
+The `elasticsearch` and `db` containers store their data to volumes, the containers will use data on these volumes if they already exist.
```

Comment:
also add note about mysql file ownership as the docker official image has uid of 999 while mysql official is 27 and thus will cause permission error on update